main.tex

```diff
@@ -316,54 +318,46 @@ \subsection*{Genetic backgrounds carrying resistance alleles}
 
 %%
 %
-Although it is known that pyrethroid resistance is increasing in prevalence in malaria vector populations across Africa \cite{Ranson2016}, it has not been clear whether this is being driven by the spread of resistance alleles via gene flow, by resistance alleles emerging independently in multiple locations, or by some combination of both processes.
-%
```

Comment:
Chopped this to keep things lean, and stay focused on inferring adaptive gene flow.